server/hydrogen-render/1-render-hydrogen-to-string.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
```

Comment:
Why the 1-4 naming? see https://github.com/matrix-org/matrix-public-archive/pull/36#discussion_r914239965